source/features/sticky-sidebar.tsx

```diff
@@ -7,8 +7,8 @@ import * as pageDetect from 'github-url-detection';
 import features from '.';
 
 const deinit: VoidFunction[] = [];
-// Both selectors are present on conversation pages so we need to discriminate
-const sidebarSelector = pageDetect.isRepoRoot() ? '.repository-content .flex-column > .flex-shrink-0 > [data-pjax]' : '#partial-discussion-sidebar';
+
+const sidebarSelector = '.Layout-sidebar > div';
```

Comment:
Both pages (repo root & conversations) have a similar layout now so we can just use a single selector \o/

Reply:
Is there a more specific selector?

Reply:
How about
```css
.Layout-sidebar > :is(.Border-grid, #partial-discussion-sidebar)
```
?

(First one in parentheses is for repo root, second one for conversation pages.)

Reply:
It looks a little better (especially with comments in the code)

Reply:
It'd be good to merge this PR by v21.11.1